spec/lib/efile/md/md502_calculator_spec.rb

```diff
@@ -1018,30 +1024,31 @@
   end
 
   describe "#calculate_line_23" do
+    let (:deduction_method_standard) { true}
+    let (:claimed_as_dependent) { false }
+
     before do
       intake.direct_file_data.fed_wages_salaries_tips = line_1b
       allow_any_instance_of(described_class).to receive(:calculate_line_7).and_return(line_7)
-      allow_any_instance_of(described_class).to receive(:deduction_method_is_standard?).and_return(true)
+      allow_any_instance_of(described_class).to receive(:deduction_method_is_standard?).and_return(deduction_method_standard)
+      allow_any_instance_of(DirectFileData).to receive(:claimed_as_dependent?).and_return claimed_as_dependent
       instance.calculate
     end
 
     context "deduction method is nonstandard" do
-      let(:line_1b) { 20_000 }
-      let(:line_7) { 15_000 }
-
-      before do
-        allow_any_instance_of(described_class).to receive(:deduction_method_is_standard?).and_return(false)
-      end
+      let(:line_1b) { 5_000 }
```

Comment:
These needed to change because the previous values made the intake ineligible for PLC - the test still passed even when I changed the filing status to not-dependent